Documentation/guides/messages/xa0125.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Xamarin.Android warning XA0125
+description: XA0125 warning code
+ms.date: 09/28/2020
+---
+# Xamarin.Android warning XA0125
+
+## Example messages
+
+```
+warning XA0125: 'AndroidApp1.pdb' is using a deprecated debug
+information level. Set the debugging information to Portable in the
+Visual Studio project property pages or edit the project file in a
+text editor and set the 'DebugType' MSBuild property to 'portable' to
+use the newer, cross-platform debug information level. If this file
+comes from a NuGet package, update to a newer version of the NuGet
+package or notify the library author.
+```
+
+## Issue
+
+Using symbol files with the `.mdb` extension or
+`<DebugType>full</DebugType>` is deprecated. Using non-portable symbol
+files with the `.pdb` extension or `<DebugType>pdbonly</DebugType>` is
+deprecated. This also includes NuGet packages or any library that
+distributes `.mdb` or non-portable `.pdb` files.
+
+## Solution
+
+`full` or `pdbonly` symbol files are a Windows-specific format and
+`<DebugType>portable</DebugType>` should be used going forward. .NET 6
+and higher will not support the older formats. Update to a newer
+version or notify the library author if you are encountering the
+`XA0125` warning from a NuGet package.
+
+In Visual Studio, go to **Properties > Build > Advanced** in the project
+property pages and change **Debugging information** to **Portable**.
+
+In Visual Studio for Mac, go to **Build > Compiler > Debug information**
+in the project property pages and change **Debug information** to
+**Portable**.
```

Comment:
Since JonP raised some finer-tooth-comb docs formatting items in PR 5165, I'll paste a few tweaks here for the MSBuild property and file name formatting to follow the docs conventions even more precisely:

````markdown
Using symbol files with the _.mdb_ extension or built with the
`DebugType` MSBuild property set to `full` is deprecated. Using
non-portable symbol files with the _.pdb_ extension or `DebugType` set
to `pdbonly` is deprecated. This also includes NuGet packages or any
library that distributes _.mdb_ or non-portable _.pdb_ files.

## Solution

`full` or `pdbonly` symbol files are a Windows-specific format, and
`DebugType` should be set to `portable` going forward. .NET 6 and higher
will not support the older formats. Update to a newer version or notify
the library author if you are encountering the XA0125 warning from a
NuGet package.

In Visual Studio, go to **Properties > Build > Advanced** in the project
property pages and change **Debugging information** to **Portable**.

In Visual Studio for Mac, go to **Build > Compiler > Debug information**
in the project property pages and change **Debug information** to
**Portable**.
````

Since I was taking another read over it, here are some possible additional wording adjustments that came to mind in case any of them look good:

````markdown
Support for _.mdb_ or _.pdb_ symbols files that were built with the
`DebugType` MSBuild property set to `full` or `pdbonly` is now
deprecated. This applies to _.mdb_ and _.pdb_ files both in application
projects and in referenced libraries, such as NuGet packages.

## Solution

Set `DebugType` to `portable` in the application project as well all
library references.  `portable` is the recommended setting for all
projects from now on.  The older `full` and `pdbonly` settings are for
older Windows-specific file formats.  .NET 6 and higher will not support
those older formats.

In Visual Studio, go to **Properties > Build > Advanced** in the project
property pages and change **Debugging information** to **Portable**.

In Visual Studio for Mac, go to **Build > Compiler > Debug information**
in the project property pages and change **Debug information** to
**Portable**.

If the problematic symbol file comes from a NuGet package, update to a
newer version of the package or notify the library author.
````